dev_suites/smart/smart_suite.rb

```diff
@@ -0,0 +1,42 @@
+module SMART
+  class SMARTSuite < Inferno::TestSuite
+    id 'smart'
+    title 'SMART'
+
+    resume_test_route :get, '/launch' do
+      request.query_parameters['iss']
+    end
+
+    group do
+      id 'smart_group'
+      title 'SMART Group'
+
+      test do
+        id 'auth_redirect'
+        title 'OAuth server redirects client browser to app redirect URI'
```

Comment:
Is this for the `launch` uri and not the `redirect` uri?  The test title doesn't quite match what i would expect you to name the request, etc.  Not a big deal, just making sure I understand the logic here.